### PR TITLE
UIOR-729 Implement Keyboard shortcuts modal

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
https://issues.folio.org/browse/UIOR-729

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by axelhunn — APPROVED** [review]

File: test/bigtest/tests/order-details-with-line-test.js [hunk 1]
```diff
@@ -17,7 +17,19 @@ import OrderDetailsPage from '../interactors/order-details-page';
 import LineDetailsPage from '../interactors/line-details-page';
 
 describe('Order details with Line', function () {
-  setupApplication();
+  const InvoiceApp = () => (<div>Invoice</div>);
```
Comment (NikitaSedyx) [inline test/bigtest/tests/order-details-with-line-test.js:20]: invoice?

File: test/bigtest/tests/line-details-test.js [hunk 2]
```diff
@@ -16,7 +16,35 @@ import LineDetailsPage from '../interactors/line-details-page';
 import OrderDetailsPage from '../interactors/order-details-page';
 
 describe('Line details test', function () {
-  setupApplication();
+  const ReceivingApp = () => (<div>Receiving</div>);
+  const AgreementsApp = () => (<div>Agreements</div>);
+  const InvoiceApp = () => (<div>Invoice</div>);
+
+  setupApplication({
+    modules: [
```
Comment (NikitaSedyx) [inline test/bigtest/tests/line-details-test.js:24]: didn't get why we need this

**Review by uladislausamets — APPROVED** [review]

File: test/bigtest/tests/line-details-test.js (hunk 2 above)
Comment (AndreiShumski2) [reply test/bigtest/tests/line-details-test.js:24]: Tests fail when redirecting to different apps  - https://jenkins-aws.indexdata.com/job/folio-org/job/ui-orders/job/PR-1133/3/#showFailuresLink

Comment (NikitaSedyx) [reply test/bigtest/tests/line-details-test.js:24]: but it was ok before, is smth changed?

Comment (AndreiShumski2) [reply test/bigtest/tests/line-details-test.js:24]: looks like it happened after `AppContextMenu` is added

**Review by NikitaSedyx — APPROVED** [review]

**Pull request merged**